controllers/dependencies.go

```diff
@@ -52,16 +56,15 @@ func (r *OperatorPipelineReconciler) reconcilePipelineDependencies(meta metav1.O
 
 				// apply pipeline yaml manifests
 				if path == PIPELINE_MANIFESTS_PATH {
-					if errors := r.applyManifests(filePath, meta.Namespace, &pipeline); errors != nil {
-						log.Error(errors, "Couldn't apply pipeline manifest")
-						return errors
+					if errs := r.applyPipelineManifests(filePath, meta, &pipeline); errs != nil {
+						log.Error(errs, "Couldn't apply pipeline manifest")
+						return errs
 					}
-
 					// or apply tasks manifests
 				} else {
```

Comment:
This else doesn't seem necessary. I think you can safely get rid of it.

Reply:
That is necessary, we have to create pipelines or tasks depending on what the directory is.

Reply:
Oh. I see it now. It returns on error only. Got it.